hydra_base/lib/project.py

```diff
@@ -92,6 +92,12 @@ def add_project(project, **kwargs):
     proj_i.name = project.name
     proj_i.description = project.description
     proj_i.created_by = user_id
+    
+    #'appdata' is a metadata column. It's not called 'metadata' because
+    #'metadata' is a reserved sqlalchemy keyword.
+    if hasattr(project, 'appdata') or project.appdata is not None:
```

Comment:
This should be "and" instead of "or". If the "hasattr(project, 'appdata')" test is false, the "project.appdata" test will raise an error